lib/handbook/addon/docs/components/link/-components/demo-external/template.hbs

```diff
@@ -0,0 +1,12 @@
+{{#docs-demo as |demo|}}
+    {{#demo.example name='link.demo-external.hbs'}}
+        <Link
+            data-analytics-name='COS link'
```

Comment:
Attributes passed without `@` are normal HTML attributes that are added as-is to the component's element. It's part of why angle-bracket invocation is awesome, you can apply things like `data-` and `aria-` attrs without the component having to be aware. (For tagless components, those attrs are [passed into the template via `...attributes`](https://github.com/CenterForOpenScience/ember-osf-web/blob/885bab8a1f140b0a635d5e9b429381807266f635/lib/osf-components/addon/components/link/template.hbs#L7).

Reply:
Quotes should be linted... the only ember-template-lint quote rule I can find is [all-double or all-single](https://github.com/CenterForOpenScience/ember-osf-web/blob/885bab8a1f140b0a635d5e9b429381807266f635/lib/osf-components/addon/components/link/template.hbs#L7)... any reason not to go all-single like the rest of the codebase?

Reply:
Because James didn't like double quotes inside of curly-brace invocation. But yeah, I tend to agree.

Reply:
Count me on team singlequote! 1️⃣

Reply:
Well, we'll have to see what @jamescdavis thinks, as I'm just passing along the feedback I got from my similar task.

Reply:
At one point (pre-angle bracket invocation) I wanted:
```hbs
<htmlelement foo="bar">
{{component-invocation param='baz'}}
{{helper 'positional_param'}}
```
(which was impossible with this rule, so I didn't enable it) because double quotes on html element attrs felt more "standard" and component/helper params felt like "code", but now either
```hbs
<htmlelement foo="bar">
<ComponentInvocation @param="baz" />
{{helper "positional_param"}}
```
or
```hbs
<htmlelement foo='bar'>
<ComponentInvocation @param='baz' />
{{helper 'positional_param'}}
```
seems fine (with a slight preference for single quotes for the reasons @aaxelb mentioned)

FYI, I looked to the guides for, uh, guidance, but they [don't even do it consistently](https://guides.emberjs.com/release/templates/links/).

Reply:
Agreed with single-quotes for the reason mentioned.

Reply:
Although apostrophes are also a thing. Is it more likely that we'd use an apostrophe in our stuff than a quote? It seems like neither would be particularly likely because internationalization.

Reply:
Yeah we probably won't (or shouldn't) use either because i18n. A concern might be quotes-in-handlebars-in-quotes like `class='Foo {{my-helper 'bar'}}'`, but I'm *pretty* sure that actually is parsed just fine.

Reply:
it is